examples/crystal.py

```diff
@@ -571,10 +611,26 @@ def plot_gaussian_fields(
     ax.set_ylim(ax.get_ylim()[::-1])
 
     plt.subplots_adjust(left=0, bottom=0, right=1, top=1)
-
     plt.savefig("crystal_gaussian.png", bbox_inches="tight")
 
 
 if __name__ == "__main__":
-    plot_dipole_fields()
-    plot_gaussian_fields()
+    # plot_dipole_fields()
```

Comment:
I presume this was for testing --- do you want to restore the calls to the plotting functions?

Reply:
Fixed, thanks